code/modules/mob/living/carbon/human/species.dm

```diff
@@ -47,6 +47,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/deathsound //used to set the mobs deathsound on species change
 	var/list/special_step_sounds //Sounds to override barefeet walkng
 	var/grab_sound //Special sound for grabbing
+	var/list/items_important_for_life = list() // A list of paths that are important for species life e.g. plasmaman internals/suit
```

Comment:
this should probably be global list(s) or something.

Reply:
actually no, we have datum outfits for this.

Reply:
plus, again, if you're going to add something like this, look at what already exists. we already have procs that handle what happens when you want to equip things, either rework those or incorporate your work similarly to keep consistent.

Reply:
Outfits replace items, and these items aren't going to really match what outfits are currently used for.

Reply:
you dont even have to use the procs, they just contain the item, and the slot to which they belong, you could make a new outfit proc if you wanted as well. The entire point is that you should ensure that your code fits whats currently in place or you make something that works better.

Reply:
You could always just expand the scope of outfits. I don't see a reason why they should not be able to add items without replacing any.

Reply:
> The entire point is that you should ensure that your code fits whats currently in place or you make something that works better.

Uh - I mean okay? I thought this PR was doing that... That's fine, I have no issue making this an outfit and adding a new proc to deal with not replacing items etc.

Reply:
^

Reply:
my point being that if you plan to make something thats works better, it has to also replace whats already there, so we don't have a ton of systems handling outfits.

Reply:
you're best off doing what nemvar just said, it'll also make the proc look a lot cleaner